### DISCUSSION TITLE
Add a central place to manage all the translations

### DISCUSSION THREAD

**Discussion by LasseRosenow** [directus/directus#20420] [Feature Requests]
### Summary

Add a central place within directus, that allows to manage all our translations that we have put somewhere deep inside of our website structure.

### Motivation

I just started working more heavily with translations, and found that the way it currently is implemented, has some problems to work with it efficiently.

The problem originates from the fact that in directus it can easily happen that we end up with a heavily nested structure of for example `pages` that have a M2A to `content`, which can be for example a `box` that has multiple `text` or `button` fields etc.

Now if we initially create our page in one language we naturally click through the menus and create all the items, but when we want to add translations later on, we end up needing to search for all the translatable fields that are `somewhere` within our `huge` structure of `tables and tables and tables containing tables and even more tables`.

### Detailed Design

Add a dedicated page to the sidebar that manages translations.
Within that page we could see all the `translation strings` that we have put somewhere in out layout and could have the ability to create translations for them.

This would also help in finding missing translations etc. and make the whole workflow much more convenient.

Instead of searching for translatable fields within our project tree we can now just go through the list and translate every string that is available there.

### Must Have:
- Central place to see all available translations
- Allow to add new translations for other languages

### Should Have:
- Inform about missing translations (using some red dots or something?)

### Could Have:
- Auto translate button using online services?

### Won't Have:
- World peace?

### Drawbacks

I don't see any drawbacks yet.

### Alternatives

No idea yet.

### Adoption Strategy

It is not a breaking change, it is a totally optional quality of life improvement.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
